index.js

```diff
@@ -61,16 +63,14 @@ function mnemonicToEntropy (mnemonic, wordlist) {
 
   var dividerIndex2 = Math.floor(bits2.length / 33) * 32
   var newChecksum2 = bits2.slice(dividerIndex2)
-
-  if (newChecksum2 !== checksum) {
-    throw new Error('Invalid mnemonic checksum')
-  }
+  if (newChecksum2 !== checksum) throw new Error('Invalid mnemonic checksum')
 
   return entropyBuffer.toString('hex')
 }
 
 function entropyToMnemonic (entropy, wordlist) {
   wordlist = wordlist || DEFAULT_WORDLIST
+  if (entropy.length % 8 !== 0) throw new TypeError('Mnemonic entropy is not a multiple of 32 bits')
```

Comment:
Is this a SEMVER issue too?
It didn't previously throw... but IMHO that was a bug.